osu.Game/Utils/ModUtils.cs

```diff
@@ -121,7 +121,7 @@ public static bool CheckValidForGameplay(IEnumerable<Mod> mods, [NotNullWhen(fal
             if (!CheckCompatibleSet(mods, out invalidMods))
                 return false;
 
-            return checkValid(mods, m => m.Type != ModType.System && m.HasImplementation, out invalidMods);
+            return checkValid(mods, m => m.HasImplementation, out invalidMods);
```

Comment:
This is required as otherwise it would be literally impossible to activate any system mod due to `OsuGame`-level checks using this method and eagerly deleting anything it didn't like.